bin/benchpark

```diff
@@ -226,6 +226,9 @@ def benchpark_setup_handler(args):
     run_command(
         f"{ramble_exe} repo add --scope=site {source_dir}/repo"
     )
+    run_command(
+        f"pip install -r {ramble_location}/requirements.txt"
+    )
```

Comment:
This would modify the user's Python installation without their knowledge (and may not work if they are using a System installation of Python). I recommend changing that:

* Add `--target` to the `pip` invocation
* Tell the user to update their `PYTHONPATH` in the below instructions that are printed

Reply:
Thanks @scheibelp! I was also leery about udpating a user's python installing without them knowing. @rfhaque had an alternative suggestion to add a flag to our ramble calls that disables the progress bar.